Objects.md

```diff
@@ -1,3 +1,46 @@
 # Objects in SuperTux
 
-Objects can be found throughout the levels in SuperTux
+Objects can be found throughout the levels in SuperTux.
+
+### Bonus Blocks
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/bonus_block/full-0.png?raw=true "Bonus Block")
+
+Bonus Blocks generally contain powerups. However, they can contain scripts and even nasty badguys. Generally you cannot see what is in the block until you hit it. They can be hit from below but also can be butt-jumped from the top.
+
+### Checkpoints
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/resetpoints/bell-m.png?raw=true "Checkpoint")
+
+Checkpoints are bells that when passed will allow you to spawn back there if you die. However, it will cost you 10% (one 10th) of your coins or 25 coins, whichever is more. If you have less than 25 coins, you will have to go back to the start.  
+
+### Doors
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/door/door-0.png?raw=true "Door")
+
+Doors can teleport you to another part of a level. Press the up key whilst in front of one to use it. Sometimes there will be a door there to go back, but not always. 
+
+### Switches
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/switch/switch-0.png?raw=true "Switch")
+
+Switches can either be levers or buttons. Levers work by pressing the up key and buttons by jumping on them. They both run scripts, which can change the level or do something to Tux. In most cases, it will add a path to make part of the level possible. Sometimes there will be two signs of the same colour at the switch and the thing that gets changed by the script. 
+
+### Platforms
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/flying_platform/flying_platform-0.png?raw=true "Platform")
+
+Platforms take Tux to and from places. Some are always on, some move when Tux gets on them and others need to be started by a switch or a script. 
+
+### Info Blocks
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/bonus_block/infoblock.png?raw=true "Info Block")
+
+Info Blocks can give valuable information about the level ahead. They can be hit from below or butt-jumped from on top. 
+
+### Trampolines
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+
+Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps.
```

Comment:
I think I mean the action key. Hah. I am getting confused with my own key setup here.